components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -906,12 +913,17 @@ public void revokeAccessTokensInBatch(String[] tokens) throws IdentityOAuth2Exce
                     ps.setString(2, UUID.randomUUID().toString());
                     ps.setString(3, getHashingPersistenceProcessor().getProcessedAccessTokenIdentifier(token));
                     ps.addBatch();
+                    oldTokens.add(getHashingPersistenceProcessor().getProcessedAccessTokenIdentifier(token));
                 }
                 ps.executeBatch();
                 connection.commit();
```

Comment:
don't we need to remove this commit()?

Reply:
yes with the revamp process "
Move JDBC transaction into structured UnitOfWork design pattern" connection.commit(); has removed